package.toml

```diff
@@ -3,10 +3,10 @@
   uri = "build/buildpack.tgz"
 
 [[dependencies]]
-  uri = "docker://quay.io/plotly/paketo-buildpacks-cpython:5.2"
+  uri = "docker://quay.io/plotly/paketo-buildpacks-cpython:5.3.0b3"
```

Comment:
I think we should use git sha an not number I approve the PR but let's try to setup this for the future 

@tarzzz

Reply:
@x3rus That makes sense to me and I think that will be captured via @antoinerg 's automation work on the buildpacks.